Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs

```diff
@@ -371,6 +378,18 @@ void ApplyIntegretionLibraries (string socialPlatform, bool remove)
 
 				EditorGUI.EndDisabledGroup();
 				break;
+			case "gameCenter":
+#if UNITY_IOS
```

Comment:
@vedi, I'm not completely sure, but add this to avoid user confusing when he will work with Android and will be impressed how we implemented GameCenter for Android (if shown, then available).